packages/cli-hermes/src/profileHermes/sourcemapUtils.ts

```diff
@@ -0,0 +1,120 @@
+import {Config} from '@react-native-community/cli-types';
+import {logger, CLIError} from '@react-native-community/cli-tools';
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import {SourceMap} from 'hermes-profile-transformer';
+import ip from 'ip';
+import nodeFetch from 'node-fetch';
```

Comment:
We have a `fetch` utility in `cli-tools`, maybe we could reuse that? If not, `node-fetch` needs to be declared in package dependencies

Reply:
That's a good idea, I changed to use the `fetch` in `cli-tools` 🚀